### ISSUE TITLE
Fail test on accessibility errors

### ISSUE THREAD

**Issue by atikenny** [chealt/chealt#64]
**Is your feature request related to a problem? Please describe.**
When testing the accessibility of the pages, I would like to make sure that the problems are fixed.

**Describe the solution you'd like**
I would like to add an option that lets me fail the test based on what level of a11y problem the page has.
